### PR TITLE
add description summary

### PR DESCRIPTION
**Pull request by mschreie1010**
where and when to be

### CODE REVIEW

File: mschreie1010-friedberg.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Far far away
```
Comment (githubteacher) [inline mschreie1010-friedberg.md:1]: Far away? Like a star? ⭐️

File: mschreie1010-friedberg.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+# long long time ago
```
Comment (githubteacher) [inline mschreie1010-friedberg.md:2]: Like yesterday?

File: mschreie1010-friedberg.md (hunk 1 above)
Comment (sssrose) [inline mschreie1010-friedberg.md:1]: Looks good!

**Review by githubteacher — DISMISSED** [review]

File: mschreie1010-friedberg.md (hunk 1 above)
Comment (EvanFishbein) [inline mschreie1010-friedberg.md:1]: Is this referring to a galaxy?

**Review by viviant100 — COMMENTED** [review]
Looks like you are getting lots of comments already!

File: mschreie1010-friedberg.md (hunk 1 above)
Comment (sssrose) [reply mschreie1010-friedberg.md:1]: so far!

**Pull request merged**